setup.py

```diff
@@ -446,6 +446,8 @@ def run(self):
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
```

Comment:
I'm not sure this should be added now as TensorFlow does not support python 3.11 yet, and I'm not too sure for PyTorch.

Reply:
that's interesting, that means I was under a wrong impression that python is backward compatible.

is it ok to keep 3.10?

Reply:
Yes 3.10 is definitely supported!